pkgs/development/libraries/gdk-pixbuf/default.nix

```diff
@@ -17,16 +17,20 @@
 , libpng
 , gnome
 , gobject-introspection
+, withIntrospection ? stdenv.buildPlatform == stdenv.hostPlatform
 , doCheck ? false
 , makeWrapper
 , lib
+, withGtkDoc ? stdenv.buildPlatform == stdenv.hostPlatform
 }:
 
 stdenv.mkDerivation rec {
   pname = "gdk-pixbuf";
   version = "2.42.6";
 
-  outputs = [ "out" "dev" "man" "devdoc" "installedTests" ];
+  outputs = [ "out" "dev" "man" ]
+    ++ lib.optional withGtkDoc "devdoc"
+    ++ lib.optional (stdenv.buildPlatform == stdenv.hostPlatform) "installedTests";
```

Comment:
Installed tests are only run at runtime (on host) so I do not see why they should not cross-compile.

Reply:
I think they're not built when cross-compiling? Will take a look, but I think I got errors that the 'installedTests' output was not created.

Reply:
error: builder for '/nix/store/q88gkc876abyx7jcvrnmb5ivdxbjb5hx-gdk-pixbuf-aarch64-unknown-linux-gnu-2.42.6.drv' failed to produce output path for output 'installedTests' at '/nix/store/q88gkc876abyx7jcvrnmb5ivdxbjb5hx-gdk-pixbuf-aarch64-unknown-linux-gnu-2.42.6.drv.chroot/nix/store/2nwmyxsr3al41xmhn4g0kpl5pf174m89-gdk-pixbuf-aarch64-unknown-linux-gnu-2.42.6-installedTests'

Reply:
Yeah, looking at the source code, they are not trivially cross-compilable. Some of the installed tests need `gdk-pixbuf-query-loaders` program to build `resources.c`.